gpii/node_modules/testing/src/CloudBasedOAuth2TestsUtils.js

```diff
@@ -184,11 +179,11 @@ gpii.test.cloudBased.oauth2.filterModel = function (model, changes) {
 };
 
 gpii.test.cloudBased.oauth2.bootstrapDisruptedTest = function (testDefs, commonRec,
```

Comment:
There's probably going to be a bit of a train wreck here when we try to merge this [with my work to convert to using CouchDB](https://github.com/GPII/universal/pull/728).  I had to change and collapse a lot of "fixture building" stuff.  As I was suggesting to @amb26, we should probably set up a pairing session for the merge.

Reply:
For context, here's what the file looks like after being converted to use sequence elements and grades:

https://github.com/the-t-in-rtf/universal/blob/GPII-3619/gpii/node_modules/testing/src/CloudBasedOAuth2TestsUtils.js

Some of the underlying functions are still used, hopefully it will be possible to reconcile the approaches without too much pain.